### ISSUE TITLE
Pyright doesn't work ctypes arrays.

### ISSUE THREAD

**Issue by kayabaNerve** [microsoft/pyright#287]
**Describe the bug**
CTypes `Array`s are created by multiplying the specific type by the size. PyRight will then throw:
```
error: Operator '*' not supported for types 'Type[c_uint64]' and 'int'
```

**To Reproduce**
Run PyRight on the following code Block.
```
from ctypes import Array, c_uint64
myArray: Array = (c_uint64 * 5)()
```

**Expected behavior**
PyRight should acknowledge this as valid code.

**Additional context**
```
$ python3 --version
Python 3.6.8
```

**Comment by erictraut** [comment]
Thanks for the bug report. You're right — the meta class for c_uint64 and related classes supports the `__mul__` method. Pyright wasn't following metaclasses for unary or binary operators. 

The fix will be in the next release of pyright.

**Comment by erictraut** [comment]
This is now fixed in 1.0.73, which I just published.

**Comment by erictraut** [comment]
@kayabaNerve I'd appreciate it if you'd create a new issue with updated code. Thanks!
